components/Timeline/TimelineEvent/TimelineEvent.tsx

```diff
@@ -1,12 +1,11 @@
-import { array, node, oneOfType, string } from 'prop-types';
 import styles from './TimelineEvent.module.css';
 
-TimelineEvent.propTypes = {
-  content: oneOfType([string, array, node]).isRequired,
-  title: string.isRequired,
+export type TimelineEventPropsType = {
+  content: string | string[] | React.ReactNode | React.ReactNode[];
```

Comment:
```suggestion
  content: React.ReactNode[];
```

I _think_ this actually covers everything, no?

Reply:
I thought so, but I was just going off the old prop-type. I need to quit being so skittish about going with my gut lol.

Reply:
I like it better if you assume everything is wrong and just try to make things right :)